pkg/platform/kube/functionres/lazy.go

```diff
@@ -225,7 +234,12 @@ func (lc *lazyClient) CreateOrUpdate(ctx context.Context, function *nuclioio.Nuc
 		return nil, errors.Wrap(err, "Failed to create/update ingress")
 	}
 
-	lc.logger.Debug("Deployment created/updated")
+	resources.cronJobs, err = lc.createCronJobsFromCronTriggers(functionLabels, function, &resources)
```

Comment:
Encapsulate cron job creation
you can simply call `createCronJobs(...)`

and within `createCronJobs` simply create all kinds of cron jobs the function needs